### ISSUE TITLE
Major issue when trying to run team talk 5.21 on macOS with voiceover

### ISSUE THREAD

**Issue by michael1994bot** [BearWare/TeamTalk5#2964]
### Description

 when trying to run the latest version of the TeamTalk client on macOS I get the following response team talk is not responding and it is very unusable with VoiceOver. I have to force quit the app and re-launch it or wait an extra 15 to 20 minutes or 10 minutes at times before the client joins also when it finally does attempt to make a connection onto a server, I am unable to navigate the channel list with my arrow keys, using voiceover neither method will work such as using quick NAV and using your up-and-down arrow keys to navigate the list. There's no mention of a channel list. There's no input being given to navigate to a specific channel no matter what input I assign or give myself as a VoiceOver user. I am unable to access my channels or any other channels for that matter on this team talk client I strongly suggest that you guys should look into this and fix the problem with the macOS client because this is not just for the latest version but this has also been previous versions before such as 18.5 and the previous version before this latest version this is a major major bug that needs to be fixed and also when turning off voiceover and turning it back on, I still get the same results so it doesn't really change the behavior of the client

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [x] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

 what I expect to happen with the team talk client on macOS is after installing the client on macOS what's supposed to happen? Is when you go and run it the client should automatically bring you directly into the team talk window where you can simply press F2 and connect to your server of choice unfortunately, when running this latest version, this particular client will not run. It just says team talk is not respondingand it is unusable.

### Steps to reproduce problem

 So recently, I have tried to close the client by force quitting the program unfortunately, that did not work. I had to restart my Mac. Secondly, I've also tried to turn off VoiceOver and turning it back on that did not solve the problem either I have tried anything and everything that I could possibly think of and unfortunately, none of my steps, and none of my strategies have worked

**Comment by ChrisDuffley** [comment]
Hi,
Regarding not being able to navigate in trees, the Speak Selected Item in Lists option has been reintroduced in Text To Speech preferences. With the first of the 2 dropdowns towards the end of the dialog set to "Default" and the second one and so on set to the language and voice you prefer, tab or VO right and you should then find a checkbox called "Speak Selected Item in Lists". Press Return and then it should start speaking what's highlighted.
As for startup, I do not have this problem. Can you please tell us what macOS you're using? I'm using macOS Sonoma if that helps clarify things.
Thanks!

**Comment by bear101** [comment]
Duplicate
#2863
